staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors.go

```diff
@@ -217,6 +217,22 @@ func NewValidatingWebhookAccessor(uid, configurationName string, h *v1.Validatin
 	return &validatingWebhookAccessor{uid: uid, configurationName: configurationName, ValidatingWebhook: h}
 }
 
+// NOTE: we're thinking about adding a new NewValidatingWebhookAccessorWithOptions to allow
```

Comment:
So this current PR makes it so new ValidatingWebhookConfigurations only make that ValidatingWebhookConfiguration recompile, we could further optimize by checking if the matchConditions actually changed and only recompile CEL if they did or even expand that to the selectors or rest comments, any thoughts on this @jpbetz @maxsmythe